ui/src/assets/docs/de/faq-page.md

```diff
@@ -0,0 +1,136 @@
+
+Der Query Parameter ist zwingend notwendig für die Durchführung. Wenn Sie nicht angefügt wird, zeigt der Redirect Server eine Standard-Fehlermeldung an.
+
+## &nbsp;
+
+**Warum kann ich den Transaktions- oder Consentstatus nicht von einem bestimmten PSU verändern**\
+Wenn der Transaktions- oder Consentstatus "received" lautet, kann es möglich sein, dass die PSU-ID nicht zur angegebenen IBAN in der Zahlungs- oder Consentanlage passt. Die definierten PSUs mit zugehörigen Testdaten finden Sie im develeoper-Portal dargestellt. <br />
+Wenn der Status sich trotz durchgefürter SCA und passendem PSU mit zugehöriger IBAN nicht verändert, achten Sie auf Typographie und Groß- und Kleinschreibung.
+
+## &nbsp;
+
+## Zertifikate
+
+**Wie kann ich ein valides Zertifikat generieren?**\
+Um auf die XS2A API zugreifen zu können, braucht ein Drittanbieter ein gültiges QWAC Zertifikat, welches zukünftig von registrierten Treuhanddienstleister (TSP=Trust Service Provider) ausgestellt wird. Um im adorsys PSD2 Accelerator Zugriff zu erhalten, können Sie sich hier ein Testzertifikat generieren:
+[https://sandbox-portal.dev.adorsys.de/app/certificate-service](https://sandbox-portal.dev.adorsys.de/app/certificate-service).
+Sie bekommen ein unterschriebenes Zertifikat (.pem) und den zugehörigen private
+key (.key) integriert in einer .zip file.
+
+## &nbsp;
+
+**Wie kann ich ein Zertifikat in einen Request integrieren?**\
+Wenn Sie ein QWAC Zertikat erhalten haben, müssen Sie es in in Ihren Request integrieren. Es gibt viele Tools, mit denen es möglich ist REST Calls durchzuführen. Wir erklären nachfolgend die Durchführung mit zwei weit verbreiteten Tools - cURL and Postman.
+cURL: Fügen Sie das .pem file mit `--cert ./certificate.p` und den private
+key mit `--key ./private.key` ein. Achten Sie darauf, dass Sie gegebenfalls den Pfand zu Ihren Dateien anpassen.
+Postman: Navigieren Sie zu "Preferences > Certificates > Add Certificate" und setzen Sie den the host url (in unserem Beispiel wäre es
+[https://sandbox-api.dev.adorsys.de](https://sandbox-api.dev.adorsys.de)).
+Achten Sie darauf, dass "https://" und der Port 443 bereits gesetzt sind. Anschließend fügen Sie .pem und .key file hinzu. Die passphrase muss dabei leer gelassen werden.
+
+## &nbsp;
+
+**Warum funktioniert mein QWAC Zertifikat nicht?**\
+Requests können gegebenefalls auch mit dem hinzugefügten Zertifikat nicht funktionieren. Das passiert, wenn Zertifikate abgelaufen sind. Jedes Zertifikat verfügt über das Attribut
+"validUntil" mit einem Ablaufdatum.
+Wenn dieser Fall zutrifft, erhalten Sie folgenden Fehler:
+
+```
+{
+  "tppMessages": [
+    {
+      "category": "ERROR",
+      "code": "CONSENT_EXPIRED",
+      "path": null,
+      "text": "The consent was created by this TPP but has expired and needs to be renewed"
+    }
+  ],
+  "_links": null
+}
+```
+
+Sie benötigen den PSU um einen neuen Consent anzulegen und anschließend diese ConsentID zu benutzen.
```

Comment:
In diesem Fall müssen Sie mit dem PSU einen neuen Consent anlegen, um die neu generierte Consent-Id benutzen zu können.

Reply:
done